scripts/doc/api-doc.ts

```diff
@@ -263,16 +285,17 @@ export function parseDocs(indexFile: string) {
 	}
 
 	function visitType(node: Node) {
+		const flags = ts.TypeFormatFlags.InTypeAlias | ts.TypeFormatFlags.NoTruncation;
 		if (ts.isFunctionLike(node) && node.name) {
 			const symbol = typeChecker.getSymbolAtLocation(node.name)!;
 			const res = typeChecker.typeToString(typeChecker.getTypeOfSymbolAtLocation(symbol, node));
 			return res;
 		}
-		if ((node as HasType)?.type) {
+		if ((node as HasType)?.type && (node as HasType).type!.getText().startsWith('SlotContent<')) {
```

Comment:
Question: what does it solve this `startWith`?

Reply:
As discussed, it was to display the type of the item transition.